federation/pkg/federation-controller/util/delaying_deliverer.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"container/heap"
+	"time"
+)
+
+const (
+	delayingDelivererUpdateChanCapacity = 1000
```

Comment:
So maybe add a TODO to figure out what the value should be (e.g. in cacher in storage we have it equal to 100 and it is enough for us - I would suspect this usecase is even less requiring).